jobserver/models.py

```diff
@@ -180,22 +206,19 @@ def runtime(self):
 
     @property
     def status(self):
-        if self.is_complete:
+        if self.is_completed:
             return "Completed"
 
-        if self.is_dependency_failed:
-            return "Dependency Failed"
-
         if self.is_failed:
```

Comment:
As each state *should be* mutually exclusive, would it make sense to make these `elifs`?

Reply:
We have Jobs which don't fit any of these states in Production, hence the fall-through to `Unknown`.

Reply:
That can still happen with `elif`?

Reply:
True.  Unless you've got strong opinions I think I'd like to leave this as is.  They're semantically identical (I believe), but I find the whitespace aids my ability to scan through the method far easier than a `elif` block.